tools/Export.mk

```diff
@@ -80,6 +80,13 @@ endif
 	@echo "OBJDUMP=\"$(OBJDUMP)\"" >> $(EXPORTDIR)/makeinfo.sh
 	@echo "OBJEXT=\"$(OBJEXT)\"" >> $(EXPORTDIR)/makeinfo.sh
 	@echo "STRIP=\"$(STRIP)\"" >> $(EXPORTDIR)/makeinfo.sh
+	@echo "NUTTX_ARCH=\"$(CONFIG_ARCH)\"" >> $(EXPORTDIR)/makeinfo.sh
```

Comment:
how about custom chip?

Reply:
I didn't known it exists in the KConfig :) I'll add it.

Reply:
so, please add it then I can merge this patch.

Reply:
done :)